pkgs/by-name/bl/blendfarm/package.nix

```diff
@@ -54,6 +55,19 @@ buildDotnetModule rec {
     hash = "sha256-2w2tdl5n0IFTuthY97NYMeyRe2r72jaKFfoNSjWQMM4=";
   };
 
+  patches = [
+    (fetchpatch {
+      url = "https://github.com/LogicReinc/LogicReinc.BlendFarm/commit/aecfb0a69f9e418158d4a09ec955e0c3f4d8f062.patch?full_index=1";
+      hash = "sha256-GaIED/TuEa2BYNNfy+/auvLzj7J7HUo2QvUq+pYJPMc=";
+    })
+    (fetchpatch {
+      url = "https://github.com/LogicReinc/LogicReinc.BlendFarm/commit/6276136de0fc567d3a2cc2ecff6effc27688d1a4.patch?full_index=1";
+      hash = "sha256-F4XW79fAMmdmDpUC0Ktt1YRpzUuMcRQp+48ryZFkSi4=";
+    })
```

Comment:
Please include the patch instead, as noted in https://github.com/NixOS/nixpkgs/tree/master/pkgs#patches. Please also write in comment what they do and tracking link.

Reply:
done

Reply:
Sorry for not being clear about that:

> Patches already merged upstream or published elsewhere should be retrieved using fetchpatch.

> ...

> Otherwise, you can add a .patch file to the nixpkgs repository. In the interest of keeping our maintenance burden and the size of nixpkgs to a minimum, only do this for patches that are unique to nixpkgs or that **have been proposed upstream but are not merged yet**, cannot be easily fetched or have a high chance to disappear in the future due to unstable or unreliable URLs. The latter avoids link rot when the upstream abandons, squashes or rebases their change, in which case the commit may get garbage-collected.

Reply:
So, to be clear, you want me to open two PRs upstream with the patches I made and fetch these?

Reply:
Ok. So this PR can be merged then, right? Or where does it not conform to the documentation? I wouldn't create a PR, since upstream is fine with net6 and we (as in nixpkgs) choose net8. So this falls iMHO under nixpkgs specific patch.

Reply:
it falls under

> is already proposed upstream but not merged yet

So the link is not persistent.